src/gd_kotlin.cpp

```diff
@@ -60,13 +60,6 @@ void register_engine_types_hook(JNIEnv* p_env, jobject p_this, jobjectArray p_en
 }
 
 void GDKotlin::init() {
-    if (Engine::get_singleton()->is_project_manager_hint()) {
```

Comment:
Why this has been removed ?

Reply:
Not removed but moved.
It is in GdjLanguage now, I just don't call GDKotlin::init() at all if in project manager.